assets/settings/default.json

```diff
@@ -797,5 +839,22 @@
   //   - `short`: "2 s, 15 l, 32 c"
   //   - `long`: "2 selections, 15 lines, 32 characters"
   // Default: long
-  "line_indicator_format": "long"
-}
+  "line_indicator_format": "long",
+  // Set a proxy to use. The proxy protocol is specified by the URI scheme.
+  //
+  // - **`http`**: Proxy. Default when no scheme is specified.
+  // - **`https`**: HTTPS Proxy. (Added in 7.52.0 for OpenSSL, GnuTLS and
+  //   NSS)
+  // - **`socks4`**: SOCKS4 Proxy.
+  // - **`socks4a`**: SOCKS4a Proxy. Proxy resolves URL hostname.
+  // - **`socks5`**: SOCKS5 Proxy.
+  // - **`socks5h`**: SOCKS5 Proxy. Proxy resolves URL hostname.
```

Comment:
We can remove `**`, the `` ` `` actually have a style in Markdown, not need to bold again.

And example should be `"proxy": "socks5://localhost:10808"`, because it is a JSON syntax.

Reply:
You are right.

Reply:
I propose to write like this:

```
// Set a proxy to use. The proxy protocol is specified by the URI scheme.
//
// By default no proxy will be used, or Zed will try get proxy settings from
// environment variables (`HTTP_PROXY`, `HTTPS_PROXY` or `ALL_PROXY`).
//
// Examples:
//
// - Use a HTTP proxy:
//
//     "proxy": "http://127.0.0.1:10809"
//
// - Use a Socket5 proxy:
//
//     "proxy": "socks5://localhost:10808"
```

We may not need to say too much about the proxy protocol. If someone need this, they will know it.

Reply:
Yes, I simplified the comments here.